torch/utils/cpp_extension.py

```diff
@@ -1086,6 +1086,10 @@ def CUDAExtension(name, sources, *args, **kwargs):
                                  hipify_result[s_abs]["hipified_path"] is not None) else s_abs)
 
         sources = list(hipified_sources)
+        relative_sources = set()
+        for source in sources:
+            relative_sources.add("." + source.replace(build_dir, ""))
+        sources = list(relative_sources)
```

Comment:
Suggest to remove the above snippet and use the snippet below for minimal changes
```suggestion
```

```
        hipified_sources = set()
        for source in sources:
            s_abs = os.path.abspath(source)
            hipified_s_abs = (hipify_result[s_abs]["hipified_path"] if (s_abs in hipify_result and
                             hipify_result[s_abs]["hipified_path"] is not None) else s_abs)
            # setup() arguments must *always* be /-separated paths relative to the setup.py directory, 
            # *never* absolute paths
            hipified_sources.add(os.path.relpath(hipified_s_abs, build_dir))

        sources = list(hipified_sources)
```

Reply:
@jithunnair-amd Thank you for your suggestion. I'll revise it here.